specs/validator/0_beacon-chain-validator.md

```diff
@@ -233,7 +233,7 @@ epoch_signature = bls_sign(
     * Let `deposit_root` and `deposit_count` be the deposit root and deposit count of the Eth 1.0 deposit contract in the post-state of the block referenced by `block_hash`
     * Let `best_vote_data = Eth1Data(block_hash=block_hash, deposit_root=deposit_root, deposit_count=deposit_count)`.
 * If `D` is nonempty:
-    * Let `best_vote_data` be the `eth1_data` of the member of `D` that has the highest `vote.vote_count`, breaking ties by favoring block hashes with higher associated block height.
+    * Let `best_vote_data` be the `eth1_data` member of `D` that has the highest vote count (`D.count(eth1_data)`), breaking ties by favoring block hashes with higher associated block height.
```

Comment:
The previous sentence "Let `D` be the set of `Eth1DataVote` objects `vote` in `state.eth1_data_votes` where: ..." makes me thought `D` is a `set`, where `D.count(eth1_data)` would always be `1`. What do you think about changing to "Let `D` be the list of `Eth1DataVote` objects `vote`"?

Reply:
You're right. will rearrange things a a bit